.murdock

```diff
@@ -26,6 +26,7 @@ mega-xplained
 microbit
 native
 nrf52840dk
+nrf59160
```

Comment:
```suggestion
nrf9160
```

Reply:
ups

Reply:
I guess I was still asleep this morning...
Should be nrf9160dk...

Reply:
done